src/ui/src/js/controllers/request_index.js

```diff
@@ -164,7 +164,12 @@ export default function requestIndexController(
       }),
     DTColumnBuilder
       .newColumn('system_version')
-      .withTitle('Version'),
+      .withTitle('Version')
+      .renderWith(function(data, type, full) {
+        let display = '';
+
+        return display + `<a ui-sref="base.namespace.system({systemName: '${full.system}', systemVersion: '${full.system_version}'})">` + data + '</a>';
```

Comment:
Is the variable here needed? Could we just do:

```javascript
return `<a ui-sref="base.namespace.system(...
```

Reply:
I made requested changes and added link for command view.